histomicstk/features/ComputeNucleiFeatures.py

```diff
@@ -167,6 +180,26 @@ def ComputeNucleiFeatures(im_label, im_nuclei, im_cytoplasm=None,
 
         feature_list.append(fgrad_cytoplasm)
 
+    # compute nuclei haralick features
+    if haralick_features_flag:
+
+        fharalick_nuclei = ComputeHaralickFeatures(im_label, im_nuclei,
```

Comment:
@cdeepakroy graycomatrixext function seems to restrict the size of nuclei box or mask to num_levels, so it occurs an error when any size of box is less than the num_levels. Do we put a condition out of the function or inside the function? And, could you inform me how to check the sparsity of the glcm?